packages/components/app/styles/components/button-set.scss

```diff
@@ -0,0 +1,13 @@
+//
+// BUTTON-SET
+//
+// properties within each class are sorted alphabetically
+//
+
+.hds-button-set {
+    display: flex;
+
+    > * + * {
+        margin-inline-start: 16px !important;
```

Comment:
[issue?] why `!important` here?

Reply:
also, what do you think of using just `margin-left` here? I know `margin-inline-start` is more correct, but the likelihood of having our apps support a different writing mode is approximate to zero :)

Reply:
I thought you wanted to basically replicate this component: https://github.com/hashicorp/structure/blob/dffbb0e0ae5246d3b832586ab25bcb981a052b30/packages/pds-ember/app/styles/pds/components/button-set/__config.scss#L10-L14

That's the only reason I included !important as I thought there was a purpose behind using it in the original component. I'll remove it.

For the margin style, I would normally just use `margin-left` unless there's a reason not to but I see that logical property margin values were used elsewhere in the component code so I assumed that was preferred.

@MelSumner Do we want to use logical properties for accessibility reasons? Or should I just use `margin-left` here?

Reply:
Maybe let's not overthink this: we can just use `margin-left`, it works and it's clear what it does.